packages/components/addon/components/hds/app-footer/legal-links.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+export default class HdsAppFooterIndexComponent extends Component {
+  get termsUrl() {
+    return (
+      this.args.termsUrl ??
+      'https://portal.cloud.hashicorp.com/terms-of-service'
```

Comment:
I think this works only for HPC/Cloud UI, so probably is not what other consumers would need. Or we have a generic company-level default, or we should make this a required argument without a default

Reply:
We could set this as the default: 
"https://www.hashicorp.com/terms-of-service"

I will change it for now and add a comment that we may want to make it required.

Reply:
@didoo Is it ok with the new default ("https://www.hashicorp.com/terms-of-service") or should I not set a default and make it required instead?

Reply:
I think is a sensible default. We can always change it later if needed, when we have a response from the Legal team